core/cap-0051.md

```diff
@@ -260,14 +260,18 @@ func $serialize_to_binary(param $val i64) (result i64)
 /// Traps if the deserialization fails for any reason.
 func $deserialize_from_binary(param $obj_bin i64, param $type u64) (result i64)
 
-/// Given a host binary object $obj_bin, copy a segment of its memory specified at $offset with length $len into guest linear memory at $pos. 
-/// Traps if either the binary object or the guest linear memory doesn't have enough bytes.
-func $binary_copy_to_guest_mem(param $obj_bin i64) (param $offset u64) (param $pos u64) (param $len u64)
+/// Given a host binary object $obj_bin, copies a segment of its memory specified at $offset with length $len into the linear memory at $pos. 
+/// Traps if either the binary object or the linear memory doesn't have enough bytes.
+func $binary_copy_to_linear_memory(param $obj_bin i64) (param $offset u64) (param $pos u64) (param $len u64)
 
-/// Copy a segment of the guest's linear memory specified at position $pos with length $len, into a host binary object $obj_bin at $offset. 
+/// Copies a segment of the linear memory specified at position $pos with length $len, into a host binary object $obj_bin at $offset. The host binary may grow in size to accommodate the new bytes. 
```

Comment:
It's unclear if this insert at offset, or fills starting at offset. Which is it? I assume it is the latter given the name is `copy` and not `insert`.

Reply:
I realize after asking it is clear that it fills, given the name.